### ISSUE TITLE
Add dropdown menu for the experimenter

### ISSUE THREAD

**Issue by alexpiet** [AllenNeuralDynamics/dynamic-foraging-task#294]
RAs request an easier way to enter their name. As part of Load mouse we should add a drop down menu for experimenters, this will ensure spelling is consistent. 
- [x] Add check for "ghost in the shell" default experimenter
- [ ] Add a drop down menu in `Load Mouse` for experimenters

**Comment by ellahiltonvano** [comment]
@smrithi-sunil per my email this morning is this an issue you could work on?

**Comment by ellahiltonvano** [comment]
Related to https://github.com/AllenNeuralDynamics/aind-behavior-blog/issues/495

**Comment by ellahiltonvano** [comment]
@micahwoodard I think there is a similar issue open but I'm not seeing it, tagging you here until I find it!

**Comment by ellahiltonvano** [comment]
It seems like we could just pull both mouse ID and experimenter names from the schedule? Alex had suggested auto populating "load mouse" = it might make more sense to just pull both of these fields and suggest them to avoid typos, or use this to generate the dropdown menu. 

https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/281

**Comment by alexpiet** [comment]
Good idea, I'll implement that. When we start a new mouse, it will grab the experimenter name from the schedule if it's available

**Comment by ellahiltonvano** [comment]
@alexpiet should we close this since its now referencing the schedule?
